cardano-cli/cardano-cli.cabal

```diff
@@ -0,0 +1,123 @@
+name:                  cardano-cli
+-- TODO @intricate: Should this match the `cardano-node` version?
+version:               0.0.1
```

Comment:
@erikd @Jimbo4350

Reply:
I think all packages in this repo should have the same version number. @dcoutts ?

Reply:
Set it to the same version as `cardano-node`.